### PR TITLE
Move tests folder to top dir level

### PR DESCRIPTION
**Pull request by bobleesj**
Before:

==== 138 passed in 11.16s =====

After:

===138 passed in 1.51s ======

Using `conftest`, local pytest runs are much faster. 

https://github.com/Billingegroup/cookiecutter/issues/114

### CODE REVIEW

**Review by sbillinge — COMMENTED** [review]
I think we just need to remove the init

**Comment by bobleesj** [comment]
@sbillinge `__init__.py` removed.

**Comment by sbillinge** [comment]
@bobleesj I am afraid that removing the init broke this.  We could add it back, but in general, if there is another approach (some small edits to the tests needed to make them pass without that init) it would be preferred so we don't have some packages differently structured from others....

**Comment by bobleesj** [comment]
@sbillinge Please review the latest change. 

CI isn't running here but checked it using my local remote branch.

Screenshot 2024-09-13 at 8 35 54 AM

**Comment by bobleesj** [comment]
> it would be preferred so we don't have some packages differently structured from others....

I am committed to this.

**Review by sbillinge — COMMENTED** [review]
Looks great! One small inline comment

File: pyproject.toml [hunk 1]
```diff
@@ -45,7 +45,7 @@ dirty_template = "{tag}"
 [tool.setuptools.packages.find]
 where = ["src"]  # list of folders that contain the packages (["."] by default)
 include = ["*"]  # package names should match these glob patterns (["*"] by default)
-exclude = ["diffpy.structure.tests*"]  # exclude packages matching these glob patterns (empty by default)
+exclude = []  # exclude packages matching these glob patterns (empty by default)
```
Comment (bobleesj) [inline pyproject.toml:48]: exclude = []  # exclude packages matching these glob patterns (empty by default)

File: pyproject.toml [hunk 2]
```diff
@@ -45,7 +45,6 @@ dirty_template = "{tag}"
 [tool.setuptools.packages.find]
 where = ["src"]  # list of folders that contain the packages (["."] by default)
 include = ["*"]  # package names should match these glob patterns (["*"] by default)
-exclude = ["diffpy.structure.tests*"]  # exclude packages matching these glob patterns (empty by default)
```
[the remark this answers is not in the document]
Comment (bobleesj) [reply pyproject.toml:48]: Got it - done.

**Comment by bobleesj** [comment]
@sbillinge 

```
exclude = []  # exclude packages matching these glob patterns (empty by default)
```

is used now. Please review.

**Pull request merged**